pages/common/tarsnap.md

```diff
@@ -0,0 +1,34 @@
+# tarsnap
+
+> Manipulate remote Tarsnap encrypted backups.
+> Note: you don't need to specify the key file and the cache directory if you configure them on `/usr/local/etc/tarsnap.conf` and/or `~/.tarsnaprc`.
```

Comment:
Only one config file is read.

https://www.tarsnap.com/man-tarsnap.1.html
>/usr/local/etc/tarsnap.conf
             The system global tarsnap configuration file.  ***Parameters
             specified here only take effect if they are not specified via the
             current user's local configuration file*** or via the command line.
```suggestion
> Note: you don't need to specify the key file and the cache directory if you configure them in `/usr/local/etc/tarsnap.conf` or `~/.tarsnaprc`.
```